sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java

```diff
@@ -39,6 +39,9 @@
     private String since;
     private String deprecated;
 
+    // The groups below are almost the same with ones defined in `sql/functions.sql`.
```

Comment:
maybe you meant `functions.sql` -> `functions.scala`?

Reply:
Ah, yes...